src/quo2/theme.cljs

```diff
@@ -43,6 +43,10 @@
   []
   (utils.transforms/js->clj (rn/use-context theme-context)))
 
+(defn use-theme-value
```

Comment:
added hook to get `theme` value directly as wrapping functional components with with-theme can be tricky.